kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -652,6 +665,17 @@ labelDel lbl = do
        else
            putStrLn' "Label doesn't exist."
 
+outputsKore :: ReplCommand -> Bool
```

Comment:
I'm not sure we need this. I would say that commands that don't output patterns should not go through pipes. We can go back on this later.